app/views/posts/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 class='font-medium text-3xl mb-8'>Create New Post</h1>
+
+<%= form_for(@post, url: user_posts_path(@user)) do |f| %>
+  <div>
+    <div>
+      <p class='font-medium'><%= f.label :title %></p>
+      <%= f.text_field :title %>
+    </div>
+    <div class='mt-8'>
+      <p class='font-medium'><%= f.label :text %><p>
+      <%= f.text_area :text %>
+    </div>  
+  </div>
+
+  <div>
+    <%= f.submit 'Create Post' class='mt-8 p-2 cursor-pointer boder-2 border-black rounded-md' %>
```

Comment:
- [ ] 👍 Great progress on building the forms for your blog app! 🚀 However, I noticed a syntax error in your post form related to the styling of the submit button. To enhance readability and resolve this issue, consider applying the styles directly to a `button` element and encapsulating it around the `f.submit` button. This ensures a cleaner structure and avoids potential syntax conflicts.

**Here's a revised version for your consideration:**

```ruby
<%= button_tag(type: 'submit', class: 'mt-8 p-2 cursor-pointer boder-2 border-black rounded-md') do %>
  Create Post
<% end %>
```

This modification separates the styling concerns, making your code more maintainable and syntactically correct.  Happy coding!😊👨‍💻🚀